conda/base/context.py

```diff
@@ -1005,6 +1005,21 @@ def log_level(self) -> int:
         else:
             return logging.WARNING  # 30
 
+    @memoizedproperty
+    def solver_user_agent(self):
```

Comment:
Is this really not covered by the unit tests?
Could we skip @memoizedproperty on this one?